ui/src/components/home/ProfileContainerMobile.tsx

```diff
@@ -0,0 +1,362 @@
+import axios from 'axios';
```

Comment:
Basically the same comments as the one in desktop lol. Ganbarre :")
But actly I think u can combine mobile and desktop tgt. See how Randal use useMediaQuery to diff mobile and desktop or u can also do what u did in the `ProfileContainer.tsx` so u don't need to duplicate files.

Reply:
the design is diff for both, so i'll prob keep the files separate, jsut bcuz there's gonan be too much state checks if i combine them to one lol :")